src/modules/ie-objects/components/ObjectDetailPageMetadata/ObjectDetailPageMetadata.tsx

```diff
@@ -507,12 +508,12 @@ export const ObjectDetailPageMetadata: FC<ObjectDetailPageMetadataProps> = ({
 		if (!isKiosk) {
 			return (
 				<div className={styles['p-object-detail__sidebar__content-maintainer-data']}>
-					{maintainerDescription && (
+					{maintainerDescription && locale === Locale.nl && (
 						<p className={styles['p-object-detail__sidebar__content-description']}>
 							{maintainerDescription}
 						</p>
 					)}
-					{maintainerSiteUrl && (
+					{maintainerSiteUrl && locale === Locale.nl && (
```

Comment:
why do we also hide the website?

Reply:
In the Jira ticket there is also a circle around the website content: https://meemoo.atlassian.net/browse/ARC-2472

Reply:
I will put the website back for now